scripts/gen_docs.sh

```diff
@@ -1,3 +1,5 @@
+# Copyright © 2022 United States Government as represented by the Administrator of the National Aeronautics and Space Administration.  All Rights Reserved.
```

Comment:
Is the year here correct? Added 2022 because of current year change was made.

Reply:
Ah, actually #!/bin/sh needs to be the first line to work correctly.

Re: the date- excellent question. I think it actually needs to be the NASA claimed the copyright on the package- but I sent an email to the software release office to confirm